driver/main.c

```diff
@@ -208,6 +208,7 @@ static const struct file_operations g_ppm_fops = {
 LIST_HEAD(g_consumer_list);
 static DEFINE_MUTEX(g_consumer_mutex);
 static u32 g_tracepoints_attached; // list of attached tracepoints; bitmask using ppm_tp.h enum
+static unsigned long per_cpu_buffer_dim = 0; // dimension of a single per-CPU buffer. We initialize it to `0` but this must be set by userspace before opening the buffers.
```

Comment:
Can we default it to whatever the old hardcoded value was? This would avoid the need for a major API version bump (just a minor one would suffice)

Reply:
Done :)